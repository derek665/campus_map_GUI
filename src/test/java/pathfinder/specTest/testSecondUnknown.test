# test correct output when the second node is unknown

CreateGraph testSecond

AddNode testSecond A

FindPath testSecond A B