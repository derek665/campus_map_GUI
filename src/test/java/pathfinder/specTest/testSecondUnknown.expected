# test correct output when the second node is unknown

created graph testSecond

added node A to testSecond

unknown node B